testing/wiremock-shaded-for-testing/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+plugins {
+  id("com.gradleup.shadow")
+  id("otel.java-conventions")
+}
+
+dependencies {
+  implementation("com.github.tomakehurst:wiremock-jre8:2.35.2")
+  implementation("com.google.errorprone:error_prone_annotations")
+  implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.18.2")
+}
+
+tasks {
+  shadowJar {
+    dependencies {
+      exclude(dependency("org.slf4j:slf4j-api"))
+      exclude(dependency("org.junit.jupiter:junit-jupiter-api"))
+      exclude(dependency("org.junit.platform:junit-platform-commons"))
+      exclude(dependency("org.ow2.asm:asm"))
+      // Exclude dependencies bundled during Armeria shading
+      exclude(dependency("com.fasterxml.jackson.core:jackson-annotations"))
```

Comment:
This is a bit weird, I wonder if instead of `x-shaded-for-testing` we should have a single `shaded-test-deps` project combining them? For this PR keeping changes to existing at a minimal though

Reply:
i agree that having a single shaded artifact might be better